concent_api/generate-requirements-lock.sh

```diff
@@ -1,11 +1,42 @@
 #!/bin/bash -e
 
+file_name="$1"
+
+
+display_help() {
+    echo "This script must be run within virtual environment"
+    echo "Script accepts exactly one parameter"
+    echo "Usage example: ./generate-requirements-lock.sh requirements.lock"
+}
+
+
+if [[  "$1" == "--help" ||  "$1" == "-h" ]]
```

Comment:
you can also print help, even when run without parameters ;)

Reply:
Added. Done :)